src/Router.cpp

```diff
@@ -12,13 +12,29 @@ Router::~Router()
 HTTPResponse Router::routeRequest(const HTTPRequest &request)
 {
 	HTTPResponse response;
-	if (isDynamicRequest(request))
+	std::string _webRoot = "var/www"; // TODO: get this from the config file
+	if (isCGI(request))
 	{
-		response = _cgiHandler.handleRequest(request);
+		response = CGIHandler().handleRequest(request);
 	}
-	else
+	else if (isDynamicRequest(request))
 	{
-		response = _staticContentHandler.handleRequest(request);
+		std::cout << "\033[31mCGI is the only dynamic requests we handle at the moment\033[0m" << std::endl;
+	}
+	else // it is a static request
+	{
+		if (!pathisValid(const_cast<HTTPRequest &>(request), response, _webRoot))
+		{
+
+			std::cout << "Path does not exist" << std::endl;
+			StaticContentHandler staticHandler;
+			response = staticHandler.handleNotFound();
+		}
+		else
+		{
+			StaticContentHandler staticContentInstance;
+			response = staticContentInstance.handleRequest(request);
+		}
 	}
```

Comment:
Can we create the CGIHandler and the StaticContentHanlder the same way? Like named or not named but the same way? I'm refering to line 20 and line 35/36

Reply:
Sure, we can